packages/edit-post/src/components/options-modal/index.js

```diff
@@ -42,6 +43,9 @@ export function OptionsModal( { isModalActive, closeModal } ) {
 				<EnableTipsOption label={ __( 'Enable Tips' ) } />
 			</Section>
 			<Section title={ __( 'Document Panels' ) }>
+				<PostLinkCheck>
```

Comment:
I'm not sure whether `PostLinkCheck` should be added here. The thing is that this option is persisted and is going to be applied to other posts. So you would expect to have this option always togglable even when it's not rendered for a given individual post.

Reply:
I was not sure if we should have the check or not. It looks strange for a user to be able to enable an option and then not see any change, but I understand the reason.
In other cases with variable support, for the Excerpt, we have the check for the discussions panel we don't have the check. I will follow your suggestion.

Reply:
It looks like changes proposed in #12394 align with your previous implementation. Let's make sure we are consistent. If we decide to add those checks in other places we should add this check back, too.